static/css/blog.css

```diff
@@ -336,14 +335,16 @@ body {
   padding-left: 0px;
   padding-right: 310px;
 }
+
 .main-inner .fauxcolumn-center-outer {
   left: 0px;
   right: 310px;
   /* IE6 does not respect left and right together */
   _width: expression(this.parentNode.offsetWidth -
-    parseInt("0px") -
-    parseInt("310px") + 'px');
+  parseInt("0px") -
```

Comment:
I'm unfamiliar with this syntax at all. Did the indentation break something?

Reply:
Hi, @mistyhacks . The syntax didn't break something, I just tested with the other syntax and revert it back as before.

.main-inner .fauxcolumn-center-outer {
        left: $(main.column.left.width);
        right: $(main.column.right.width);
        /* IE6 does not respect left and right together */
        _width: expression(this.parentNode.offsetWidth -
            parseInt("$(main.column.left.width)") -
            parseInt("$(main.column.right.width)") + 'px');
      }